tests/test_features_creator.py

```diff
@@ -1 +1,40 @@
-from features_creator import features_creator
+from features_creator.features_creator import get_matching_column_names
```

Comment:
This is part of the hacky workaround mentioned in #27 I hope that I will be able to change this back to `from features_creator import get_matching_column_names` tomorrow.